Annotation/polar_golden_set.json

```diff
@@ -470,7 +507,19 @@
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "Whether the judge will agree with Rocco",

```

Comment:
I think this is not a fronted embedded clausal complement of "depends on" but actually just a clausal subject

Reply:
yes, agree, I remember this one thinking "mmmm"